service/routes.py

```diff
@@ -81,57 +81,55 @@ def create_wishlists():
     ),
     status.HTTP_201_CREATED
   )
-
+  
 ######################################################################
 # List ALL WISHLISTS
 ######################################################################
 @app.route("/wishlists", methods=["GET"])
 def list_all_wishlists():
   """
-  Lists all wishlists
+  Lists all wishlists 
```

Comment:
You can tweak your VS code settings to automatically delete trailing spaces when you save a file. Please enable and remove these spaces.
<img width="1052" alt="Screen Shot 2021-11-10 at 12 47 18 PM" src="https://user-images.githubusercontent.com/33898347/141165935-2d17058b-b8aa-46c8-a31f-a0ad0370e568.png">